### PR TITLE
Allow verify and change commands in RepoUtil to work cross plat

### PR DESCRIPTION
**Pull request by karajas**
Quick fix to workaround this issue before the shift over to 2.0

The way I have this setup in my build is to inject a RuntimeIdentifier (probably need to add a default for win7-x64) and publish a stand alone app. I could make this a lot neater but wasn't sure how much effort should go into this before moving over to the 2.0 tooling.

/cc @ellismg @jaredpar

### CODE REVIEW

File: src/Tools/RepoUtil/RepoUtil.csproj [hunk 1]
```diff
@@ -1,86 +1,14 @@
...
-    <UpdatePeriodically>false</UpdatePeriodically>
-    <UpdateRequired>false</UpdateRequired>
-    <MapFileExtensions>true</MapFileExtensions>
-    <ApplicationRevision>0</ApplicationRevision>
-    <ApplicationVersion>1.0.0.%2a</ApplicationVersion>
-    <IsWebBootstrapper>false</IsWebBootstrapper>
-    <UseApplicationTrust>false</UseApplicationTrust>
-    <BootstrapperEnabled>true</BootstrapperEnabled>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <PackageId>RepoUtil</PackageId>
```
Comment (ellismg) [inline src/Tools/RepoUtil/RepoUtil.csproj]: I think the default is to use the project-name so you may not need to set PackageId specifically.

File: src/Tools/RepoUtil/RepoUtil.csproj [hunk 2]
```diff
@@ -1,86 +1,14 @@
...
-    <UpdateIntervalUnits>Days</UpdateIntervalUnits>
-    <UpdatePeriodically>false</UpdatePeriodically>
-    <UpdateRequired>false</UpdateRequired>
-    <MapFileExtensions>true</MapFileExtensions>
-    <ApplicationRevision>0</ApplicationRevision>
-    <ApplicationVersion>1.0.0.%2a</ApplicationVersion>
-    <IsWebBootstrapper>false</IsWebBootstrapper>
-    <UseApplicationTrust>false</UseApplicationTrust>
-    <BootstrapperEnabled>true</BootstrapperEnabled>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
```
Comment (ellismg) [inline src/Tools/RepoUtil/RepoUtil.csproj]: @jaredpar Would you rather we cross compile this for net46 as well so you don't need to run it on core exclusively?

File: src/Tools/RepoUtil/ChangeCommand.cs [hunk 3]
```diff
@@ -202,7 +202,7 @@ private void ChangeGeneratedFiles()
             var msbuildData = _repoData.RepoConfig.MSBuildGenerateData;
             if (msbuildData.HasValue)
             {
-                var fileName = new FileName(_generateDirectory, msbuildData.Value.RelativeFilePath);
+                var fileName = new FileName(_generateDirectory, msbuildData.Value.RelativeFilePath.Replace("\\", "/"));
```
Comment (ellismg) [inline src/Tools/RepoUtil/ChangeCommand.cs]: This feels like the wrong place to do the normalization (assuming it is needed)?

File: src/Tools/RepoUtil/RepoUtil.csproj (hunk 1 above)
Comment (karajas) [inline src/Tools/RepoUtil/RepoUtil.csproj]: Will fix.

File: src/Tools/RepoUtil/ChangeCommand.cs (hunk 3 above)
Comment (karajas) [inline src/Tools/RepoUtil/ChangeCommand.cs]: thought this would be the case. will fix.

File: src/Tools/RepoUtil/RepoUtil.csproj (hunk 2 above)
Comment (karajas) [inline src/Tools/RepoUtil/RepoUtil.csproj]: I think the current build requires msbuild on the sln file along with some logic in Settings.props and Imports.targets which I've sort of avoided. Let me know if cross compilation is the goal.

File: src/Tools/RepoUtil/RepoUtil.csproj [hunk 4]
```diff
@@ -1,86 +1,17 @@
...
-    <UpdateIntervalUnits>Days</UpdateIntervalUnits>
-    <UpdatePeriodically>false</UpdatePeriodically>
-    <UpdateRequired>false</UpdateRequired>
-    <MapFileExtensions>true</MapFileExtensions>
-    <ApplicationRevision>0</ApplicationRevision>
-    <ApplicationVersion>1.0.0.%2a</ApplicationVersion>
-    <IsWebBootstrapper>false</IsWebBootstrapper>
-    <UseApplicationTrust>false</UseApplicationTrust>
-    <BootstrapperEnabled>true</BootstrapperEnabled>
+    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
```
Comment (jaredpar) [inline src/Tools/RepoUtil/RepoUtil.csproj]: Can't take this change.  This opts the project into the new SDK + project system.  This is a much bigger work item that I'm doing in a separate branch.  Need to keep this project PCL for time being.

File: src/Tools/RepoUtil/RepoUtil.csproj [hunk 5]
```diff
@@ -1,86 +1,17 @@
...
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'" />
+
   <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Xml" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="WindowsBase" />
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="System.Collections.Immutable" Version="1.3.1" />
```
Comment (jaredpar) [inline src/Tools/RepoUtil/RepoUtil.csproj]: Same feedback.

File: src/Tools/RepoUtil/RepoUtil.csproj (hunk 4 above)
Comment (karajas) [inline src/Tools/RepoUtil/RepoUtil.csproj]: This change mostly keeps everything else the same from before but adds a new csproj for the new tooling. I can work around this now. Should I go ahead and close this pr out?

**Pull request closed without merging**